lib/routes/bins/delete.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+var debug = require('debug-log')('mockbin')
+
+module.exports = function (req, res, next) {
+  this.client.del(['bin:' + req.params.uuid, 'log:' + req.params.uuid], function (err, value) {
+    if (err) {
+      debug(err)
+
+      throw err
+    }
```

Comment:
I see this conditional
```
if(err) {
    debug(err);
    throw err;
}
```
repeated in several files. Is it possible to add an error handler function that does the check and debug&throws if need be?
```
function errorCheck(err){
    if (err) { debug(err); throw err; }
}

this.client.del(['bin:' + req.params.uuid, 'log:' + req.params.uuid], function (err, value) {
    errorCheck(err);
    // Rest of Function
}
```